src/frontend/EdFi.DataManagementService.Frontend.AspNetCore/AspNetCoreFrontend.cs

```diff
@@ -64,7 +64,7 @@ string dmsPath
     {
         if (frontendResponse.LocationHeaderPath != null)
         {
-            string urlBeforeDmsPath = httpContext.Request.UrlWithPathSegment()[..^dmsPath.Length];
+            string urlBeforeDmsPath = httpContext.Request.UrlWithPathSegment()[..^dmsPath.Length].TrimEnd('/');
```

Comment:
I was wrong - .NET does _not_ have a URI equivalent to `path.Join(..)`. Closest thing [is this constructor](https://learn.microsoft.com/en-us/dotnet/api/system.uri.-ctor?view=net-8.0#system-uri-ctor(system-uri-system-string)): `public Uri (Uri baseUri, string? relativeUri);`.

`.TrimEnd('/')` _should_ work.